bundles/org.openhab.binding.novafinedust/src/main/java/org/openhab/binding/novafinedust/internal/sds011protocol/SDS011Communicator.java

```diff
@@ -253,7 +252,7 @@ public void serialEvent(SerialPortEvent event) {
                 reply = readReply();
                 logger.debug("Got data from sensor: {}", reply);
             } catch (IOException e) {
-                logger.error("Could not read available data from the serial port", e);
+                logger.warn("Could not read available data from the serial port", e);
```

Comment:
No need to log the stacktrace here, but you could log the exception message:
```suggestion
                logger.warn("Could not read available data from the serial port: {}", e.getMessage());
```

Reply:
Indeed, this makes sense to only log the message. I have changed it, thanks!